game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyFrame.java

```diff
@@ -115,26 +112,9 @@ public ChatMessagePanel getChatMessagePanel() {
                     "Please consult other admins before banning longer than 1 day. \n"
                         + "And please remember to report this ban.",
                     date -> {
-                      controller.banMac(clickedOn, date);
+                      controller.banUser(clickedOn, date.toInstant());
```

Comment:
I wonder if it would make sense to migrate `TimespanDialog` to `Instant`, now that there are some callers that actually use it, currently the class internally uses Instants, but converts them to dates for convenience.